source/Makefile

```diff
@@ -1,153 +1,177 @@
 # This is the Makefile of ABACUS
-
-ifdef GNU
-    include Makefile.gnu
+include Makefile.vars
+#==========================
+# Compiler information 
+#==========================
+INCLUDES = -I. -Icommands -I../
+LIBS = -lifcore -lm -lpthread
+OPTS = -Ofast -march=native -std=c++14 -pedantic -m64 ${INCLUDES}
+HONG = -D__LCAO
+OBJ_DIR = obj
+BIN_DIR = ../bin
+ifeq ($(findstring mpi, $(CC)), mpi)
+     # We do not support EXX in serial version temporarily.
+     HONG += -D__MPI -D__EXX -DEXX_H_COMM=2 -DUSE_CEREAL_SERIALIZATION -DEXX_DM=3 -DEXX_H_COMM=2 -DTEST_EXX_LCAO=0 -DTEST_EXX_RADIAL=1
+     MPI = ON
+     TESTNP = 4
+     suffix = mpi
+     ifeq ($(findstring mpii, $(CC)), mpii)
+         INTEL=ON
+     endif
 else
-    include Makefile.system
+     TESTNP = 1
+     suffix = serial
+     ifeq ($(findstring i, $(CC)), i)
+         INTEL=ON
+     endif
 endif
-include Makefile.Objects
 
-VPATH=./src_global\
-:./module_orbital\
-:./module_neighbor\
-:./module_symmetry\
-:./module_cell\
-:./module_base\
-:./module_md\
-:./module_pw\
-:./module_surchem\
-:./module_deepks\
-:./module_xc\
-:./module_esolver\
-:./module_hsolver\
-:./module_hsolver/genelpa\
-:./module_elecstate\
-:./module_psi\
-:./module_hamilt\
-:./module_hamilt/ks_pw\
-:./module_gint\
-:./src_pw\
-:./src_lcao\
-:./module_relaxation\
-:./src_io\
-:./src_parallel\
-:./src_pdiag\
-:./src_pdiag/MRRR\
-:./src_ri\
-:./\
+ifeq ($(DEBUG), ON)
+    INTEL = OFF
+    HONG += -D__DEBUG
```

Comment:
Where was this flag defined in code? Defining `NDEBUG` is standarized.

Reply:
It is not used now. However, we should define a HONG for debugging and maybe it is not the name "__DEBUG"